toml.md

```diff
@@ -417,8 +422,9 @@ str = ''''That,' she said, 'is still pointless.''''
 ```
 
 Control characters other than tab are not permitted in a literal string. Thus,
-for binary data, it is recommended that you use Base64 or another suitable ASCII
-or UTF-8 encoding. The handling of that encoding will be application-specific.
+for binary data, it is recommended that you use Base64 or another suitable
+binary-to-text encoding. The handling of that encoding will be
+application-specific.
```

Comment:
Nit: I’d prefer the wording “implementation specific”. Not all implementations will be applications (in the common meaning of that word, ie, libraries are not applications, usually).

Reply:
I think we should just strike the entire last sentence. TOML does not handle the topic of binary encodings. Let those who do, figure all that out themselves. They don't need us saying anything.

Reply:
Sounds good